pages/news/2020-05-22-why-mixxx-will-not-support-streaming.html

```diff
@@ -0,0 +1,84 @@
+- The freedom to distribute copies of your modified versions to others. By doing this you can give the whole community a chance to benefit from your changes. Access to the source code is a precondition for this.
+
+Supporting streaming services would require taking away the first of those freedoms: the freedom to run the program as you wish, *for any purpose*.
+The terms of service for using any streaming service in a software application require that the application does not record or broadcast the streamed music.
+Recording and broadcasting are features of Mixxx, so complying with these terms would mean that we would have to intentionally prevent you from using Mixxx to do what it was designed to do.
+
+Every streaming service has these restrictive terms and always will have these terms in the future.
+The streaming business model depends on this.
+If users could save downloaded content from streaming services, they would not be streaming services; they would be music stores.
+There would be no purpose of paying a subscription fee to continue to access music that you have already downloaded without imposing this artificial restriction prohibiting you from saving the music.
+Hypothetically, if there was a streaming service in the future that did not have these restrictions, there would be little benefit of supporting streaming from it in Mixxx because you would be able to download music files from it like any other online music store.
+
+To be clear, there is nothing in Mixxx's license, the [GNU General Public License](https://www.gnu.org/licenses/old-licenses/gpl-2.0.html), which would legally prevent us from complying with the terms of a streaming service.
+Free software is not about laws; it is about freedom.
+Even if you do not use the recording or broadcasting features of Mixxx, or you personally would agree to the restrictive terms of a streaming service, we want to maintain your trust that
+**Mixxx will always work for you, the user. Mixxx will never work for a company against your wishes.**
+
+Some users and Mixxx developers argue that a user should be able to agree to the restrictive terms of a streaming company if they want.
+You can still do that with other software, but that does not mean we have to support it in Mixxx.
+Mixxx does not have to do everything. It does not mix videos, nor control lights, nor does it brew coffee.
+Mixxx has always been a DJ application for mixing music files on your computer and that is what it will continue to be.
+
+Besides the restrictive terms of streaming services, there are many more reasons to not support streaming in Mixxx.
+
+Because everyone has the freedom to change how Mixxx works, it would be easy to disable restrictions on recording and broadcasting from streaming services.
+If a user user didn't want to do this on their own, it would be easy for anyone  to do it for them and host downloads of modified versions of Mixxx that could record and broadcast from streaming services.
```

Comment:
```suggestion
If a user didn't want to do this on their own, it would be easy for anyone to do it for them and host downloads of modified versions of Mixxx that could record and broadcast from streaming services.
```